cmake/Modules/FindBLAS.cmake

```
@@ -20,7 +20,7 @@ SET(BLAS_INCLUDE_DIR)
 SET(BLAS_INFO)
 SET(BLAS_F2C)
 
-SET(WITH_BLAS "" CACHE STRING "Blas type [mkl/open/goto/acml/atlas/accelerate/veclib/generic]")
+SET(WITH_BLAS "" CACHE STRING "Blas type [blis/mkl/open/goto/acml/atlas/accelerate/veclib/generic]")
```

Can we sort library names alphabetically?
Also, shouldn't their capitalization match that of `BLAS` property? 
```suggestion
SET(WITH_BLAS "" CACHE STRING "Blas type [accelerate/acml/atlas/blis/generic/goto/mkl/open/veclib]")
```

---

Sorted them as suggested, however retained blis in small letters to match that of mkl's convention. BLAS is BLIS/MKL and WITH_BLAS is blis/mkl